### PR TITLE
[WIP] Adding a logo to communities via ActiveStorage (#38)

### PR DESCRIPTION
**Pull request by cwant**
If you look at this, don't forget to do a "rake db:migrate".

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,6 +22,9 @@ source 'https://rails-assets.org' do
   gem 'rails-assets-tether', '>= 1.3.3'
 end
 
+gem 'activestorage'
+gem 'font-awesome-rails'
```
Comment (murny) [inline Gemfile]: probably merge error, font awesome is included already on L13: https://github.com/ualbertalib/jupiter/blob/d5618abf2d79195d438a6f7f2beb384eda83dd5a/Gemfile#L13

Comment (cwant) [inline Gemfile]: Yep, cheers, I guess I wasn't careful enough fixing the merge conflicts.

File: app/assets/stylesheets/communities.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.community-logo-small.img-thumbnail {
+  height: 100px;
+}
```
Comment (cwant) [inline app/assets/stylesheets/communities.scss]: Funny that hound catches this, rubocop doesn't.

Comment (murny) [inline app/assets/stylesheets/communities.scss]: hound comes with a bunch of other linters other then rubocop (like haml, sass, eslint, etc). Rubocop only checks ruby.

**Comment by cwant** [comment]
I'm closing this PR and opening another to review a more up-to-date (and rebased) branch.

**Pull request closed without merging**